### DISCUSSION TITLE
What does this mean?

### DISCUSSION THREAD

**Discussion by 9wow** [oobabooga/text-generation-webui#5663] [Q&A] [answered]
When I load a model it says this below 'Successfully loaded':
"It seems to be an instruction-following model with template "Custom (obtained from model metadata)". In the chat tab, instruct or chat-instruct modes should be used."

Videos I see of people loading models just says successfully loaded without that extra part.

The models I'm using weren't installed using the web ui, maybe that's why?

**Comment by Alsefina** [comment, marked as answer]
The extra part just tells you which instruction template and chat mode should be used. You can change the instruction template in the Parameters tab and the chat mode can be set in the chat tab. The instruction template is important, make sure you set that to whatever template the model understands. The chat mode is less important from my experience, and it depends on what you want to do with the mode.
And yes, I get that message too with every model I load, its more odd if you dont.
